static/safari-pinned-tab.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
This has some unexpectedly sharp edges:
![image](https://user-images.githubusercontent.com/2085013/49683982-0248cb00-fa9b-11e8-99a4-d1cc996494c7.png)
Here are color and B&W SVGs that I cut from the larger KoBoToolbox logo:
[kobo-svg.zip](https://github.com/kobotoolbox/kpi/files/2659518/kobo-svg.zip)
Also, I'm not a Safari user, but should the interior negative space be transparent? I'd think it should remain white (and that's what I did in the attached SVGs).

Reply:
I'm also not a Safari user 😛 According to [docs](https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariWebContent/pinnedTabs/pinnedTabs.html) the icon should be single color, as they recolor all nontransparent layers. Also they require a single path (I made a compound path).